schema_version: 1.2.0
id: CVE-2016-2513
aliases: []
modified: 2017-09-08T01:29Z
published: 2016-04-08T15:59Z
details: The password hasher in contrib/auth/hashers.py in Django before 1.8.10 and
  1.9.x before 1.9.3 allows remote attackers to enumerate users via a timing attack
  involving login requests.
severity:
- type: CVSS_V3
  score: 3.1
- type: CVSS_V2
  score: 2.6
affected:
- package:
    name: django
    cpe: cpe:2.3:a:djangoproject:django
- ranges:
  - type: GIT
    repo: https://github.com/django/django
    events:
    - introduced: '0'
    - fixed: 67b46ba7016da2d259c1ecc7d666d11f5e1cfaab
- versions:
  - '1.9'
  - 1.9.2
  - 1.8.9
  - 1.9.1
- database_specific:
    CWE:
    - CWE-200
    CVSS:
      Score: 3.1
      Severity: LOW
      Code: CVSS:3.0/AV:N/AC:H/PR:N/UI:R/S:U/C:L/I:N/A:N
references:
- type: ADVISORY
  url: http://www.debian.org/security/2016/dsa-3544
- type: WEB
  url: http://rhn.redhat.com/errata/RHSA-2016-0506.html
- type: WEB
  url: https://www.djangoproject.com/weblog/2016/mar/01/security-releases/
- type: WEB
  url: http://www.ubuntu.com/usn/USN-2915-2
- type: WEB
  url: http://rhn.redhat.com/errata/RHSA-2016-0504.html
- type: WEB
  url: http://www.ubuntu.com/usn/USN-2915-1
- type: WEB
  url: http://www.securitytracker.com/id/1035152
- type: WEB
  url: http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html
- type: WEB
  url: http://rhn.redhat.com/errata/RHSA-2016-0505.html
- type: FIX
  url: https://github.com/django/django/commit/67b46ba7016da2d259c1ecc7d666d11f5e1cfaab
- type: WEB
  url: http://rhn.redhat.com/errata/RHSA-2016-0502.html
- type: WEB
  url: http://www.ubuntu.com/usn/USN-2915-3
- type: WEB
  url: http://www.securityfocus.com/bid/83878
